org.scummvm.ScummVM.json

```diff
@@ -32,11 +71,22 @@
                     "url": "https://github.com/scummvm/scummvm/archive/v2.0.0.tar.gz",
                     "sha256": "1a7cb080d956220657734ac42f75168552171bd13983d58c4bd07fdf31bcc59a"
                 },
+                {
+                    "type": "patch",
+                    "path": "fluidsynth.patch"
+                },
                 {
                     "type": "file",
                     "path": "org.scummvm.ScummVM.appdata.xml"
                 }
             ],
+            "config-opts": [
+                "--enable-release",
+                "--with-fluidsynth-prefix=/app"
+            ],
+            "build-commands": [
+                "sed -i s:Exec=scummvm:Exec=/app/bin/scummvm_wrapper: dists/scummvm.desktop"
```

Comment:
`Exec=scummvm_wrapper` should be sufficient. The `/app/bin` directory is already in PATH.